test/parallel/test-eventtarget.js

```diff
@@ -685,3 +685,37 @@ let asyncTest = Promise.resolve();
     et.dispatchEvent(new Event('foo'));
   });
 }
+
+{
+  const et = new EventTarget();
+
+  throws(() => et.addEventListener(), {
+    code: 'ERR_MISSING_ARGS',
+    name: 'TypeError',
+    message: 'The "type" and "listener" arguments must be specified'
```

Comment:
We tend not to validate error message as they are not covered by semver-rules.
```suggestion
```

Reply:
I can remove validating error message after landing #45676.

Reply:
Rebased this PR and remove validating error message in test case.